packages/manager/apps/pci-private-registry/src/pages/create/steps/NameStep.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  OsdsFormField,
+  OsdsInput,
+  OsdsText,
+} from '@ovhcloud/ods-components/react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { ODS_INPUT_TYPE, ODS_TEXT_SIZE } from '@ovhcloud/ods-components';
+import { StepComponent } from '@ovh-ux/manager-react-components';
+import { useTranslation } from 'react-i18next';
+import { StepEnum } from '@/pages/create/types';
+import { useStore } from '@/pages/create/store';
+
+export default function NameStep() {
+  const { t: tCreate } = useTranslation('create');
+  const { t: tCommon } = useTranslation('common');
+  const { t: tCommonField } = useTranslation('common_field');
```

Comment:
i guess both way have some downsides since in your example you will have to use namespaces along with the key right ?

since we adopted this way of using translations everywhere in the app but also in the other pci migration's app we won't do your suggestion here but i can understand your point